chapters/ch09.md

```diff
@@ -18,24 +18,24 @@ Giả sử bạn quan tâm đến cả độ chính xác lẫn thời gian chạ
 
 > It seems unnatural to derive a single metric by putting accuracy and running time into a single formula, such as:
 
-Việc tạo ra một phép đo đơn trị bằng cách đưa cả độ chính xác và thời gian chạy vào trong một công thức có vẻ không tự nhiên, ví dụ như:
+Việc tạo ra một phép đo đơn trị bằng cách gộp độ chính xác và thời gian chạy vào trong cùng một công thức khá là gượng ép, chẳng hạn:
 
 > Accuracy - 0.5*RunningTime
 
 Độ chính xác - 0.5*(Thời gian chạy)
 
 > Here’s what you can do instead: First, define what is an "acceptable" running time. Lets say anything that runs in 100ms is acceptable. Then, maximize accuracy, subject to your classifier meeting the running time criteria. Here, running time is a "satisficing metric"—your classifier just has to be "good enough" on this metric, in the sense that it should take at most 100ms. Accuracy is the "optimizing metric."
 
-Thay vào đó, bạn có thể làm như sau: Trước hết định nghĩa thế nào là một mốc thời gian chạy "chấp nhận được". Giả sử mốc dưới 100ms là chấp nhận được. Sau đó, hãy cực đại hóa độ chính xác, với ràng buộc là bộ phân loại đó vẫn đảm bảo yêu cầu về thời gian chạy. Ở đây, thời gian chạy là một "phép đo thỏa mãn" -- bộ phân loại của bạn chỉ cần "đủ tốt" về mặt này (thời gian), theo nghĩa nó chỉ được phép chạy trong thời gian ít hơn 100ms. Độ chính xác mới là "phép đo tối ưu".
+Thay vào đó, bạn có thể làm như sau: Trước hết hãy định nghĩa thời gian chạy như thế nào là "chấp nhận được". Ta giả sử mốc dưới 100ms là chấp nhận được. Sau đó hãy cực đại hóa độ chính xác, với ràng buộc là bộ phân loại vẫn đảm bảo yêu cầu về thời gian chạy. Ở đây, thời gian chạy là một "phép đo thỏa mãn" —- bộ phân loại của bạn chỉ cần "đủ tốt" trên thang đo đó, tức chỉ tốn tối đa 100ms để chạy xong. Độ chính xác mới là "phép đo để tối ưu".
```

Comment:
- Thay vào đó, bạn có thể ~~làm như sau~~
- Mình nghĩ có dấu "phẩy" sau từ "Trước hết" và "sau đó" được đặt ở đầu câu thì mới đúng chính tả. Mà không chắc chắn lắm :D 
- Có thể gộp 2 câu vào cho gọn 
> Trước hết, hãy định nghĩa thời gian chạy như thế nào là "chấp nhận được" - ví dụ mức 100ms.
-  mình nghĩ không có dấu "phẩy" ở đây. 
> Sau đó hãy cực đại hóa độ chính xác, với ràng buộc là bộ phân loại vẫn đảm bảo yêu cầu về thời gian chạy.
- Bỏ từ "của bạn"
>  bộ phân loại ~~của bạn~~ chỉ cần "đủ tốt"
- this --> này
> đủ tốt" trên thang đo ~~đó~~ này,

Reply:
```suggestion
Thay vào đó, bạn có thể: Trước hết, hãy định nghĩa thời gian chạy như thế nào là "chấp nhận được" - ví dụ mức dưới 100ms. Sau đó hãy cực đại hóa độ chính xác, với ràng buộc là bộ phân loại vẫn đảm bảo yêu cầu về thời gian chạy. Ở đây, thời gian chạy là một "phép đo thỏa mãn" —- bộ phân loại chỉ cần "đủ tốt" trên thang đo này, tức chỉ tốn tối đa 100ms để chạy xong. Độ chính xác mới là "phép đo để tối ưu".
```